reference/docs-conceptual/dsc/resources/authoringResourceComposite.md

```diff
@@ -183,6 +181,28 @@ configuration RenameVM
 }
 ```
 
+You can also use this resource to create multiple VMs by passing in an array of VM names to the xVirtualMachine resource.
+
+```PowerShell
+    Configuration MultipleVms
```

Comment:
Can you remove the tab of the code sample by one?  It's spaced right and doesn't need to be.